plugins/community/neoeinstein-prost/v0.3.1/buf.plugin.yaml

```diff
@@ -16,6 +16,7 @@ registry:
       - name: "prost"
         req: "0.12.3"
         default_features: false
+        features: ["std"]
       # prost-types is necessary for any module using the WKTs.
       - name: "prost-types"
```

Comment:
Should we do the same here? https://docs.rs/prost-types/0.13.1/prost_types/#feature-flags